Compentio.Example.WebAPI/AutofacModule.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Autofac;
+using Compentio.Example.WebAPI.Repository;
+using Compentio.Example.WebAPI.Services;
+using Compentio.SourceMapper.DependencyInjection;
+
+namespace Compentio.Example.WebAPI
```

Comment:
- Lets use naming `Compentio.Example.Web.Api` for assembly: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-assemblies-and-dlls and corresponding namespaces.
This name also should give us a possibility to add new example assembly for `StructureMap` DI also.
The names should be consistent and self explained.

- What is a reason of adding `Web Api` here? One project is Console App, another Web Api and for StructureMap will be.. ?

Reply:
I will change this to Compentio.Example.Web.Api.
I needed to create extra project due to Autofac requirement. We need to use Startup file with IServiceCollection and ContainerBuilder - can't do this in console App. I couldn't run Autofac on existing example. 
Can we use all 3 container DI in one example project? Maybe we should change naming like Compentio.Example.Autofac.App, Compentio.Example.Structuremap.App, but now I made comments in Program files in example projects.

Reply:
Yes, maybe `Compentio.Example.Autofac.App, Compentio.Example.Structuremap.App`, we should use